velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -146,6 +146,43 @@ class StreamingAggregationTest : public OperatorTestBase {
             "SELECT c0, max(c1 order by c2), max(c1 order by c2 desc), array_agg(c1 order by c2) FROM tmp GROUP BY c0");
   }
 
+  void testDistinctAggregation(
+      const std::vector<VectorPtr>& keys,
+      uint32_t outputBatchSize = 1'024) {
```

Comment:
perhaps, remove default value to make it easier to read the code (no need to guess what the value is)